float64[] pose
float64[] vel
float64[] ang
float64[] angRate
